config/jobs/cert-manager/cert-manager/release-previous/cert-manager-release-previous-presubmits.yaml

```diff
@@ -45,9 +43,6 @@ presubmits:
     # still, we might as well run it just in case
     - release-1.8
     - release-1.7
-    annotations:
-      testgrid-create-test-group: 'false'
-      description: Verifies dependency related files are up to date
```

Comment:
The 'description' annotation was there to provide a description for the job in the Testgrid UI, if we don't add the job to the UI no need for this annotation